Stripe/STPIDEALBankSelectorDataSource.m

```diff
@@ -30,11 +30,11 @@ - (instancetype)init {
     self = [super init];
     if (self) {
         _bankNameToBankCode = @{
-                                @"ABN AMRO": @"abn_amro",
```

Comment:
I think we can leave brand names out of the localization pipeline. I think we should also not localize payment method names that are brand names like `SOFORT` and `Giropay` – I'll update.